checker.go

```diff
@@ -15,6 +16,23 @@ func cleanMsg(m map[string]interface{}) map[string]interface{} {
 	if _, ok := m["context"].(map[string]interface{}); ok {
 		m["context"] = delete(m["context"].(map[string]interface{}), "library")
 	}
+	// delete empty fields
+	for k, v := range m {
+		if reflect.TypeOf(v).String() == "string" {
```

Comment:
Is it desirable to consider empty string to be same as non-existent?

If so  - can we also add a test for this path?

Reply:
good catch, empty strings should not be equivalent to non-existent now that I think about it. Will change.